ext/pdo_firebird/tests/pdofirebird_001.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+PDO_firebird subclass basic
+--EXTENSIONS--
+pdo_firebird
+--SKIPIF--
+<?php require(__DIR__ . '/skipif.inc'); ?>
+--FILE--
+<?php
+
+require_once __DIR__ . "/testdb.inc";
+
+$db = getDbConnection();
+
+$db->query('RECREATE TABLE pdofirebird_001 (idx int NOT NULL PRIMARY KEY, name VARCHAR(20))');
```

Comment:
This is the same as `DROP TABLE` followed by `CREATE TABLE`.

I think `CREATE TABLE` is fine because the test has cleanup.

```suggestion
$db->query('CREATE TABLE pdofirebird_001 (idx int NOT NULL PRIMARY KEY, name VARCHAR(20))');
```